src/webportal/src/app/job/job-view/fabric/job-detail/components/task-role-container-list.jsx

```diff
@@ -446,21 +482,47 @@ export default class TaskRoleContainerList extends React.Component {
             className={c(t.h100, t.flex, t.justifyStart, t.itemsCenter, t.ml1)}
           >
             <div className={c(t.flex)} style={{ height: 40 }}>
-              {config.launcherType !== 'k8s' && (
-                <CommandBarButton
-                  className={c(FontClassNames.mediumPlus)}
-                  styles={{
-                    root: { backgroundColor: 'transparent' },
-                    rootDisabled: { backgroundColor: 'transparent' },
-                  }}
-                  iconProps={{ iconName: 'CommandPrompt' }}
-                  text='View SSH Info'
-                  onClick={() => this.showSshInfo(item.containerId)}
-                  disabled={
-                    isNil(item.containerId) || item.taskState !== 'RUNNING'
+              <CommandBarButton
+                className={c(FontClassNames.mediumPlus)}
+                styles={{
+                  root: { backgroundColor: 'transparent' },
+                  rootDisabled: { backgroundColor: 'transparent' },
+                }}
+                iconProps={{ iconName: 'CommandPrompt' }}
+                text='View SSH Info'
+                onClick={() => {
+                  let hasUserSsh = false;
+                  const jobConfig = this.context.jobConfig;
+                  if (
+                    'extras' in jobConfig &&
+                    'com.microsoft.pai.runtimeplugin' in jobConfig.extras
+                  ) {
+                    for (const pluginSetting of jobConfig.extras[
+                      'com.microsoft.pai.runtimeplugin'
+                    ]) {
+                      if (pluginSetting.plugin === 'ssh') {
+                        if (
+                          'parameters' in pluginSetting &&
+                          'userssh' in pluginSetting.parameters &&
+                          !isEmpty(pluginSetting.parameters.userssh)
+                        ) {
+                          hasUserSsh = true;
+                          break;
+                        }
+                      }
+                    }
```

Comment:
why not put this logic into `showSshInfo` function?

Reply:
Done. Better to have it in `showSshInfo`.